homeassistant/components/nut/sensor.py

```diff
@@ -46,6 +46,13 @@
     "serial": ATTR_SERIAL_NUMBER,
 }
 
+AMBIENT_PRESENT = "ambient.present"
+AMBIENT_SENSORS = [
+    "ambient.humidity",
+    "ambient.humidity.status",
+    "ambient.temperature",
+    "ambient.temperature.status",
+]
```

Comment:
```suggestion
AMBIENT_SENSORS = {
    "ambient.humidity",
    "ambient.humidity.status",
    "ambient.temperature",
    "ambient.temperature.status",
}
```

Since each iteration is doing a lookup its faster to make this a set https://wiki.python.org/moin/TimeComplexity